src/app/services/main-form-manager/config-map-export-helper.ts

```diff
@@ -35,19 +36,56 @@ export class ConfigMapExportHelper {
 
     public static process(mapConfigLayers: FormArray, colorService: ArlasColorGeneratorLoader, taggableFields?: Set<string>) {
 
-        const layers: Array<Layer> = mapConfigLayers.controls.map((layerFg: FormGroup) => {
-            return this.getLayer(layerFg, colorService, taggableFields);
+        const layers: Array<[Layer, LAYER_MODE]> = mapConfigLayers.controls.map((layerFg: FormGroup) => {
+            return [this.getLayer(layerFg, colorService, taggableFields), layerFg.value.mode as LAYER_MODE];
         });
+        let layersHover: Array<[Layer, LAYER_MODE]> = mapConfigLayers.controls.map((layerFg: FormGroup) => {
+            return [this.getLayer(layerFg, colorService, taggableFields), layerFg.value.mode as LAYER_MODE];
+        });
+        let layersSelect: Array<[Layer, LAYER_MODE]> = mapConfigLayers.controls.map((layerFg: FormGroup) => {
+            return [this.getLayer(layerFg, colorService, taggableFields), layerFg.value.mode as LAYER_MODE];
+        });
+
+        layersHover = layersHover
```

Comment:
I think we can declare 
```
const layersHover = layers.filter...
```
and 
```
const layersSelect = layers.filter...
```

directly to avoid doing `mapConfigLayers.controls.map` 3 times